WeCantSpell.Hunspell/FlagSet.cs

```diff
@@ -506,44 +454,30 @@ public bool DoesNotContainAny(FlagSet other)
                 : (
                     _values!.Length == 1
                         ? other.DoesNotContain(_values[0])
-                        : !SortedInterectionTest(other._values, _values)
+                        : !SortedInterectionTest(other._values.AsSpan(), _values.AsSpan())
```

Comment:
Sorted intersection is generic enough it may be worth extracting it to one of the helper static classes.

Reply:
Nah. It is only used in specific builds too.